packages/widgetbook/lib/src/fields/duration_field.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+
+import 'field.dart';
+import 'field_codec.dart';
+import 'field_type.dart';
+
+class DurationField extends Field<Duration> {
+  DurationField({
+    required super.name,
+    required Duration super.initialValue,
+    super.onChanged,
+  }) : super(
+          type: FieldType.doubleInput,
```

Comment:
We need to create a new `FieldType` enum member called `duration`.
This `type` is used to make Fields easily renderable by external systems (like Widgetbook Cloud)

Reply:
Oh was unaware of that. Thought since the duration is in ms, it would fit as a doubleInput.

Would make the change.